airflow/config_templates/config.yml

```diff
@@ -463,6 +463,15 @@
       type: string
       example: ~
       default: "task"
+    - name: creds_backend
+      description: |
+        Classes to use for creds backend
+      version_added: ~
+      type: string
+      example: ~
+      default: >-
+        airflow.creds.environment_variables.EnvironmentVariablesCredsBackend,
+        airflow.creds.metastore.MetastoreCredsBackend
```

Comment:
Why call it creds/credentials backend when it's a backend for connections? And in case credentials is the preferred wording, why not write it out in full (credentials vs creds)?

Reply:
so... i struggled with this a bit myself.

i can call it `BaseConnectionBackend` if that's the general consensus -- not firmly attached to _creds_.

as to why though, i wrestled with it a bit myself.  but i guess i thought creds was a more apt description of what this does.  this is not really fetching connections per se but connection _meta_ or connection _info_ or plainly _creds_.  it is a backend for getting creds from arbitrary creds store, and it provides a way to yield connections from those creds.

and why creds instead of _credentials_? 

when you can have economy _and_ clarity at the same time, why not?

Reply:
I'd prefer Credentials - but it is not very strong preference.